src/view/reviewManager.ts

```diff
@@ -1024,6 +1027,65 @@ export class ReviewManager implements vscode.DecorationProvider {
 		await this.validateState();
 	}
 
+	public async createPullRequest(): Promise<void> {
+		const HEAD = this._repository.state.HEAD;
+		if (!HEAD.upstream) {
```

Comment:
`upstream` seems to always be undefined

Reply:
discussed offline, the local branch needs to be `--set-upstream` to the remote one.